cmd/postcli/main.go

```diff
@@ -416,23 +406,25 @@ func loadKey() (ed25519.PublicKey, error) {
 func cmdVerifyPos(opts config.InitOpts, fraction float64, logger *zap.Logger) {
 	log.Println("cli: verifying", edKeyFileName)
 	pub, err := loadKey()
-	if err != nil {
+	switch {
+	case errors.Is(err, os.ErrNotExist):
+		log.Println("cli:", edKeyFileName, "does not exist. Skipping verification")
```

Comment:
The message is a little misleading. It skips cross-checking the keys in the identity key file and metadata JSON file, not verifying the data. Perhaps just don't print anything?
```suggestion
```